packages/web/src/pages/profile-page/ProfilePageProvider.tsx

```diff
@@ -238,7 +238,6 @@ class ProfilePage extends PureComponent<ProfilePageProps, ProfilePageState> {
     if (!profile) return
     const userId = profile.user_id
     this.props.onUnfollow(userId)
-    this.props.setNotificationSubscription(userId, false)
```

Comment:
unfollow already dispatches a `SET_NOTIFICATION_SUBSCRIPTION` action. this was causing 2 unsubscribe requests to be issued to identity